CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# How To Contribute
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
+We welcome contributions from external members of the team! We encourage you to float the idea for your PR first, via a GitHub issue or a post in Slack.
+
+## Figma export-go instructions
+
+- For export-go, the Figma export tool, see [export-go/README](export-go/README.md).
+
+- After every new Figma export, we need to manually copy the icons to the `flight-icons/icons` directory right now.
+
+```bash
+rm flight-icons/icons/*
+```
+
+```bash
+cp export-go/dist/* flight-icons/icons
+```
+
+## sprite instructions
+
+- To generate the sprite file, [sprite.svg](ember-flight-icons/public/icons/sprite.svg), follow the instructions in the README of [edenspiekermann/sprite.sh](https://github.com/edenspiekermann/sprite.sh) e.g.
+
+```bash
+cd flight-icons/icons
+```
+
+```bash
+npx spritesh
+```
+
+The `spitesh` command will great a `sprite.svg`. Copy this file to ember-flight-icons/
+
+```bash
+cd .. && cp flight-icons/icons/sprite.svg ember-flight-icons/public/icons && cp flight-icons/icons/_catalog.json ember-flight-icons/public/icons
+```
+
+## npm instructions
```

Comment:
with this PR, i've already published the packages to test that the instructions were correct (before merge to `main`, took that liberty since we are on beta and needed a lot of debugging to get the instructions right). it would be cool if we had someone else follow them next publish, so that we can see what does and does not make sense!